src/types.ts

```diff
@@ -47,3 +47,10 @@ export interface App {
 export interface AppOptions {
   debug?: boolean
 }
+
+export interface RuntimeError {
+  statusCode: number
+  statusMessage: string
+  body?: any
+  runtime?: boolean
```

Comment:
If interface is called `RuntimeError` maybe makes sense to set type to `runtime: true` ?

Reply:
In this case, `createError` will be dedicated to runtime errors. For internal errors, users must use other methods?  
Should rename the interface to a more generic name, like `HttpError`

Reply:
Indeed since internal errors require inspecting original error object to get status, etc. We might have `InternalError` type and both extending `HttpError`